cli/tauri-bundler/src/bundle/settings.rs

```diff
@@ -682,7 +687,30 @@ impl Settings {
     self.bundle_settings.exception_domain.as_ref()
   }
 
-  /// Copies external binaries to a path.
+  pub fn osx_signing_identity(&self) -> Option<&String> {
+    self.bundle_settings.osx_signing_identity.as_ref()
+  }
+
+  pub fn osx_entitlements(&self) -> Option<&String> {
+    self.bundle_settings.osx_entitlements.as_ref()
+  }
+
+  #[cfg(target_os = "windows")]
+  pub fn windows_digest_algorithm(&self) -> Option<&String> {
+    self.bundle_settings.windows_digest_algorithm.as_ref()
+  }
+
+  #[cfg(target_os = "windows")]
+  pub fn windows_certificate_thumbprint(&self) -> Option<&String> {
+    self.bundle_settings.windows_certificate_thumbprint.as_ref()
+  }
+
+  #[cfg(target_os = "windows")]
+  pub fn windows_timestamp_url(&self) -> Option<&String> {
+    return self.bundle_settings.windows_timestamp_url.as_ref();
+  }
```

Comment:
Why are the windows attributes conditionally compiled but the macos ones aren't?

Reply:
because the windows stuff uses some APIs that won't compile on unix systems, so everything is conditionally compiled.